knimin/lib/util.py

```diff
@@ -198,6 +198,26 @@ def categorize_age(x):  # noqa
     return age_cat
 
 
+def correct_age(age, height, weight, etoh):
+    """Infers incorrect ages and incorrectly classified babies"""
+    # Checks the logic for age
+    if age >= 4 and age < 123:
```

Comment:
those are the ages we are making sure are correct through the height, weight, and alcohol checks.